src/index.html

```diff
@@ -1,16 +1,50 @@
 <!doctype html>
 <html lang="en">
+
   <head>
```

Comment:
```suggestion
<html lang="en">
  <head>
```
Don't add newlines between parent and child elements